### DISCUSSION TITLE
Could not find milvus.yaml after building milvus2.4

### DISCUSSION THREAD

**Discussion by Molly-Xu00** [milvus-io/milvus#31913] [Q&A and General discussion]
I built Milvus standalone2.4, but I didn't find milvus.yaml. I just followed the instruction of install Milvus. The files after I start milvus are as follows.
<img width="193" alt="image" src="https://github.com/milvus-io/milvus/assets/65323198/b029153d-abf5-4077-8c57-c2cfb5a8eb0a">

**Comment by yhmo** [comment]
Build milvus from source code:
```
# Clone github repository.
$ git clone https://github.com/milvus-io/milvus.git

# Install third-party dependencies.
$ cd milvus/
$ ./scripts/install_deps.sh

# Compile Milvus.
$ make
```

After build finished, you will see ./bin/milvus under the project root path.
Steps to start a local milvus:

1. cmd to the project root path
2. source ./scripts/setenv.sh
3. use a text tool to open the ./config/milvus.yaml, replace all the "/var/lib/milvus" to be "/tmp/milvus"
4. cd deployments/docker/dev
5. docker-compose up -d
6. cd -
7. ./scripts/start_standalone.sh

Step 2 is to set environment path
Step 3 is to avoid the permission deny for some OS during milvus runtime, the ./config/milvus.yaml is used to start the local milvus.
Step 4/5 is to start the third-party services, etcd/minio
